app/src/main/res/values/dimens.xml

```diff
@@ -59,6 +59,6 @@
 
     <!-- In HomeTileScreenshotStore, we generate screenshots of this size and scale them:
          it should be a fine estimate for now. -->
-    <dimen name="home_tile_icon_estimated_width">480px</dimen>
-    <dimen name="home_tile_icon_estimated_height">270px</dimen>
+    <dimen name="home_tile_icon_estimated_width">50dp</dimen>
```

Comment:
I see that the placeholder *does not* fill the space we give it (i.e. the same space as a screenshot). Are these numbers being used as absolute values (i.e. it shouldn't be scaled up or down)? In that case, this is no longer an estimate: please rename these fields and update the comment to be more accurate to their current usage (and mention that they're meant to be used absolutely).

On that note, we probably only need one of these if it's a square.